test/integrationtests.jl

```diff
@@ -890,6 +890,68 @@ end
     end
 end
 
+@testset "Backtraces timeout trigger" begin
+    function gdb_available()
+        try
+            run(`gdb -ex "exit"`)
```

Comment:
is there a way to run CI on machines with `gdb` available? why don't the linux CI machines have it?

Reply:
yeah, i think that just needs an `if:` line like
```
- name: Install GDB
  if: matrix.os == 'ubuntu-latest'
  run: sudo apt-get install -y gdb
```
not certain on the syntax, might need to be
```
- name: Install GDB
  if: ${{ matrix.os }} == 'ubuntu-latest'
  run: sudo apt-get install -y gdb
```

Reply:
I added it, but the CI machine does not allow `gdb` to attach to the process:
```
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Inappropriate ioctl for device.
```
So I've backed it out. 🤷‍♂️